ha/virsh/tests/resource_ipaddr2_test.sh

```diff
@@ -14,23 +14,22 @@ oneTimeSetUp() {
 }
 
 configure_cluster_properties() {
-  run_command_in_node "${IP_VM01}" "sudo crm configure property stonith-enabled=off"
+  run_command_in_node "${IP_VM01}" "sudo pcs property set stonith-enabled=false"
 }
 
 configure_ipaddr2_resource() {
   NAME="${1}"
   IP="${2}"
   MASK="${3}"
-  run_command_in_node "${IP_VM01}" "sudo crm configure primitive ${NAME} ocf:heartbeat:IPaddr2 \
-	  ip=${IP} cidr_netmask=${MASK} op monitor interval=30s"
+  run_command_in_node "${IP_VM01}" "sudo pcs resource create ${NAME} ocf:heartbeat:IPaddr2 \
+	  ip=${IP} cidr_netmask=${MASK} op monitor interval=30s --wait"
```

Comment:
I was checking the `pcs` manpage again, and `--wait` without a value defaults to 1h. Maybe we should set a value closer to the `sleep` values you had before? Perhaps 60s, to give it some margin? Not just here, but in all `--wait` calls